elixir/apps/web/lib/web/auth.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Web.Auth do
   use Web, :verified_routes
-  alias Domain.Auth
+  alias Domain.{Auth, Accounts}
 
   def signed_in_path(%Auth.Subject{actor: %{type: :account_admin_user}} = subject) do
     ~p"/#{subject.account.slug || subject.account}/sites"
```

Comment:
Could this be simplified to just `~p"/#{subject.account}/sites"` now that you have a Param impl for Account?

Reply:
Good catch!